### PR TITLE
(FM-7218) Bump to java test version on windows.

### PR DESCRIPTION
**Pull request by david22swan**
Also includes PDK update ran against experimental changes to revert the appveyor unit test’s to the standard spec tests.

### CODE REVIEW

File: spec/spec_helper_acceptance.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def create_keys_for_test(host)
   # Generate private key and CA for keystore
   if host['platform'] =~ %r{windows}i
     temp_dir = 'C:\\tmp\\'
-    on host, 'mkdir /cygdrive/c/tmp'
+    on host, 'mkdir -p /cygdrive/c/tmp'
```
[the remark this answers is not in the document]
Comment (david22swan) [reply spec/spec_helper_acceptance.rb]: Ah, that is needed when rerunning test's on a saved machine. Without it they error out and fail.

File: .travis.yml [hunk 2]
```diff
@@ -45,9 +45,9 @@ matrix:
       rvm: 2.1.9
 branches:
   only:
-    - master
-    - /^v\d/
-    - release
+    - master
```
[the remark this answers is not in the document]
Comment (david22swan) [reply .travis.yml]: Have removed them, but they look template added

File: spec/spec_helper_acceptance.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (david22swan) [reply spec/spec_helper_acceptance.rb]: Removed

**Comment by pmcmaw** [comment]
LGTM. Recommend running through adhoc as it touches windows testing as Appveyor doesn't run acceptance tests on Windows.

**Comment by david22swan** [comment]
ran green on adhoc

**Pull request merged**